boards.txt

```diff
@@ -1318,6 +1318,111 @@ d1_mini.menu.FlashSize.4M1M.build.spiffs_end=0x3FB000
 d1_mini.menu.FlashSize.4M1M.build.spiffs_blocksize=8192
 d1_mini.menu.FlashSize.4M1M.build.spiffs_pagesize=256
 
+##############################################################
+# d1_mini_lite ESP8285 chip has built-in 1MB flash
+
+d1_mini_lite.name=Wemos D1 mini lite (ESP8285)
+
+d1_mini_lite.upload.tool=esptool
+d1_mini_lite.upload.speed=460800
+d1_mini_lite.upload.resetmethod=nodemcu
+d1_mini_lite.upload.maximum_size=1044464
+d1_mini_lite.upload.maximum_data_size=81920
+d1_mini_lite.upload.wait_for_upload_port=true
+d1_mini_lite.serial.disableDTR=true
+d1_mini_lite.serial.disableRTS=true
+
+d1_mini_lite.build.mcu=esp8266
+d1_mini_lite.build.f_cpu=80000000L
+d1_mini_lite.build.board=ESP8266_ESP01
```

Comment:
This isn't working for me. I replaced the boards.txt file with this file yet "Wemos D1 mini lite (ESP8285)" doesn't pop up in my boards on the Arduino IDE.

Reply:
It is merged. Have a try with master. Restart the IDE after updating.